### DISCUSSION TITLE
User 100000 own lot of files/directory

### DISCUSSION THREAD

**Discussion by Kalimger** [AdguardTeam/AdGuardHome#6624] [Q&A]
Hi everyone,

I have some questions regarding a user with ID 100000 on my AdGuard Home that seems to own a large number of files and generates errors/warnings such as "Detected unsafe path transition / (owned by 100000) -> /run (owned by root) during canonicalization of."

I checked all users by typing "getent passwd" to see if there is a user with ID 100000, but it seems that the user 100000 doesn't exist...

As I am relatively new to Linux, I'm wondering if this is normal or if I missed something during the installation.

Note: I installed my AdGuard Home in an LXC container on a Proxmox server.

Thanks in advance for your help and advice!

**Comment by ainar-g** [comment]
Seems to be an [LXC thing](https://kcore.org/2022/02/05/lxc-subuid-subgid/)?
